### PR TITLE
BROKERS: Get All Groups

### PR DESCRIPTION
**Pull request by evangelinedrink**

### CODE REVIEW

File: Taarafo.Portal.Web/Brokers/Apis/ApiBroker.Groups.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Taarafo.Portal.Web.Models.Groups;
+
+namespace Taarafo.Portal.Web.Brokers.Apis
+{
+    public partial class ApiBroker
+    {
+        private const string GroupsRelativeUrl = "api/groups";
+        public async ValueTask<List<Group>> GetAllGroupsAsync() =>
```
Comment (ShriHumrudha) [inline Taarafo.Portal.Web/Brokers/Apis/ApiBroker.Groups.cs:15]: New line break required

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Do you need to register the broker?

File: Taarafo.Portal.Web/Brokers/Apis/ApiBroker.Groups.cs (hunk 1 above)
Comment (evangelinedrink) [reply Taarafo.Portal.Web/Brokers/Apis/ApiBroker.Groups.cs:16]: I added a line break in the code rub titled "CODE RUB: Added Line Break."

**Comment by evangelinedrink** [comment]
Is registering the broker pushing the broker into the main? I have worked on the Foundation Services for Retrieve All Groups, but I haven't created a Pull Request because the Model for Groups isn't in the Foundation Services branch.

**Comment by ShriHumrudha** [comment]
> Do you need to register the broker?

@evangelinedrink This is still pending

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**